bootstrapper-maven-plugin/src/main/resources/templates/Reconciler.java

```diff
@@ -0,0 +1,55 @@
+package {{groupId}};
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.ConfigMapBuilder;
+import io.fabric8.kubernetes.api.model.ObjectMetaBuilder;
+import io.javaoperatorsdk.operator.api.config.informer.InformerConfiguration;
+import io.javaoperatorsdk.operator.api.reconciler.*;
```

Comment:
{ nitpick } maybe we can avoid star imports.

Reply:
done.